src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {
+    public String getRandomColor() {
+        String[] colors = {"red", "blue", "green", "yellow", "white"};
+        Random random = new Random();
```

Comment:
Think about which variables should be local in the method and which should be class-level. [Click](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level).